js/scripts.js

```diff
@@ -0,0 +1,107 @@
+var phones,
+    btnRing = document.getElementsByClassName('btn-ring'),
+    btnText = document.getElementsByClassName('btn-text'),
+    btnPhoto = document.getElementsByClassName('btn-photo'),
+    paragraphs = document.getElementsByTagName('p');
+
+// ---------------------------------------------------------------------------------------------------
+
+function Phone(brand, model, price, color, screenSize, resolH, resolV) {
```

Comment:
za duzo parametrow to tez niedobrze, latwo pomylic ich kolejnosc. Mozesz sprobowac przekazywac jeden argument ktory bedzie obiektem:
```
{
  model: ...
  brand: ...
}
```
i wtedy kolejnosc argumentow nie bedzie miala znaczenia.
Co wiecej mozesz tez poczytac o http://2ality.com/2015/01/es6-destructuring.html#parameter-handling i sprobowac tego uzyc ale ostrzegam, moze nie byc latwo :)